/**:
  ros__parameters:
    input_channels:
      detected_objects:
        topic: "/perception/object_recognition/detection/objects"
        optional:
          name: "detected_objects"
          short_name: "all"
      # LIDAR - rule-based
      lidar_clustering:
        topic: "/perception/object_recognition/detection/clustering/objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: false
          can_trust_extension: false
          can_trust_classification: false
          can_trust_orientation: false
        optional:
          name: "clustering"
          short_name: "Lcl"
      # LIDAR - DNN
      lidar_centerpoint:
        topic: "/perception/object_recognition/detection/centerpoint/objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: true
          can_trust_extension: true
          can_trust_classification: true
          can_trust_orientation: true
        optional:
          name: "centerpoint"
          short_name: "Lcp"
      lidar_centerpoint_validated:
        topic: "/perception/object_recognition/detection/centerpoint/validation/objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: true
          can_trust_extension: true
          can_trust_classification: true
          can_trust_orientation: true
        optional:
          name: "centerpoint"
          short_name: "Lcp"
      lidar_apollo:
        topic: "/perception/object_recognition/detection/apollo/objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: true
          can_trust_extension: true
          can_trust_classification: true
          can_trust_orientation: true
        optional:
          name: "apollo"
          short_name: "Lap"
      lidar_apollo_validated:
        topic: "/perception/object_recognition/detection/apollo/validation/objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: true
          can_trust_extension: true
          can_trust_classification: true
          can_trust_orientation: true
        optional:
          name: "apollo"
          short_name: "Lap"
      # LIDAR-CAMERA - DNN
      # cspell:ignore lidar_pointpainting pointpainting
      lidar_pointpainting:
        topic: "/perception/object_recognition/detection/pointpainting/objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: true
          can_trust_extension: true
          can_trust_classification: true
          can_trust_orientation: true
        optional:
          name: "pointpainting"
          short_name: "Lpp"
      lidar_pointpainting_validated:
        topic: "/perception/object_recognition/detection/pointpainting/validation/objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: true
          can_trust_extension: true
          can_trust_classification: true
          can_trust_orientation: true
        optional:
          name: "pointpainting"
          short_name: "Lpp"
      # CAMERA-LIDAR
      camera_lidar_fusion:
        topic: "/perception/object_recognition/detection/clustering/camera_lidar_fusion/objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: false
          can_trust_extension: false
          can_trust_classification: true
          can_trust_orientation: false
        optional:
          name: "camera_lidar_fusion"
          short_name: "CLf"
      # CAMERA-LIDAR+TRACKER
      detection_by_tracker:
        topic: "/perception/object_recognition/detection/detection_by_tracker/objects"
        flags:
          can_spawn_new_tracker: false
          can_trust_existence_probability: false
          can_trust_extension: false
          can_trust_classification: false
          can_trust_orientation: false
        optional:
          name: "detection_by_tracker"
          short_name: "dbT"
      # RADAR
      radar:
        topic: "/sensing/radar/detected_objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: true
          can_trust_extension: false
          can_trust_classification: true
          can_trust_orientation: false
        optional:
          name: "radar"
          short_name: "R"
      radar_far:
        topic: "/perception/object_recognition/detection/radar/far_objects"
        flags:
          can_spawn_new_tracker: true
          can_trust_existence_probability: true
          can_trust_extension: false
          can_trust_classification: true
          can_trust_orientation: false
        optional:
          name: "radar_far"
          short_name: "Rf"
